### PR TITLE
New smtp provider

### PR DESCRIPTION
**Pull request by tramuntanal**
New SMTP provider  requires changing the SMTP configuration.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 source "https://rubygems.org"
 
 ruby RUBY_VERSION
-DECIDIM_VERSION = { git: "https://github.com/CodiTramuntana/decidim", branch: "release/0.27-stable" }
+DECIDIM_VERSION = { path: "/home/oliver/prog/decidim/fork_coditramuntana-0_27" }
```
Comment (laurajaime) [inline Gemfile:6]: weeeee!

**Comment by tramuntanal** [comment]
Thanks @laurajaime!
I've applied all your comments

**Comment by laurajaime** [comment]
> @tramuntanal you must change CI Ruby version too!

you missing that!

**Review by laurajaime — APPROVED** [review]
Good!

**Pull request merged**